backend/baseclass.pm

```diff
@@ -881,7 +879,7 @@ sub check_asserted_screen {
             _reduce_to_biggest_changes($failed_screens, 20);
         }
     }
-    bmwqemu::diag("no match $n");
+    bmwqemu::fctwarn("no match $n");
```

Comment:
warn is a too high level for "no match" which can appear many times. Shouldn't warn be reserved for events that should not happen and the user should care about it when he sees the warning more than once?